src/App.jsx

```diff
@@ -14,57 +15,57 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {value ? (
+        <h1 className="title is-flex is-align-items-center">
+          {value} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setValue(null)}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            const isValueEqual = good === value;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={isValueEqual ? 'has-background-success-light' : ''}
+                key={goods.indexOf(good) + 1}
```

Comment:
instead of indexOf, let's use index from map method
Because If you have two "Jam", the key will be the same

Reply:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/92045702/ae6e883b-8385-4dcb-a7df-8d6127583b8d)
We can't use index from map method because of linter rules

Reply:
![image](https://github.com/mate-academy/react_goods-selector-js/assets/168312293/16bfb1bf-0a02-4864-bf0e-714f033ef3d2)
We can't use index from map method because of linter rules

Reply:
Yes, but let's move it to the variable 
![image](https://github.com/mate-academy/react_goods-selector-js/assets/46566190/f0bb8fa8-8e46-40a7-a18d-197d48078c96)